src/activate/activate.c

```diff
@@ -385,6 +396,11 @@ static int parse_argv(int argc, char *argv[]) {
                 return -EINVAL;
         }
 
+        if (arg_datagram && arg_accept) {
+                log_error("Datagram sockets do not accept connections. Use -a option only with stream sockets.");
```

Comment:
I think this would sound a lot clearer if the sentence would say "The --datagram and --accept options may not be combined."